src/hooks/useCommands.js

```diff
@@ -0,0 +1,21 @@
+import Fuse from 'fuse.js'
+
+const useCommands = (commands, precision = 0.4) => {
+	commands = !!commands
+		? Object.entries(commands)?.reduce((acc, [key, value]) => ({ [key.toLocaleLowerCase()]: value }), {})
+		: {}
+
+	const triggerCommand = (input) => {
+		const fuse = new Fuse(Object.keys(commands), { includeScore: true, ignoreLocation: true })
+		const result = fuse.search(input).filter((r) => r.score < precision)
+		if (!!result?.length) {
+			const key = result[0].item.toLocaleLowerCase()
```

Comment:
Why `.toLocaleLowerCase()`? As in, why lowercase, and why specifically locale lowercase?

Reply:
Commands are just keys, we don't care about case.
You're right about the `locale`, it is more an habit than anything else.